devel/raja/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           mpi 1.0
+
+github.setup        LLNL RAJA 2022.10.5 v
+name                raja
+revision            0
+categories          devel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         RAJA Performance Portability Layer (C++)
+long_description    RAJA is a library of C++ software abstractions, primarily developed \
+                    at Lawrence Livermore National Laboratory (LLNL), that enables architecture \
+                    and programming model portability for HPC applications.
+
+fetch.type          git
+
+if {${os.major} < 14} {
+    # Lion+ (with Xcode 4.1+) have git; earlier need to bring their own.
+    # However Lion through Mavericks fail with SSL errors.
+    depends_build-append \
+                    port:git
+    git.cmd         ${prefix}/bin/git
+}
+
+set py_ver          3.11
+set py_ver_nodot    [string map {. {}} ${py_ver}]
+depends_build-append \
+                    port:pkgconfig \
+                    port:python${py_ver_nodot}
+depends_lib-append  port:cppcheck
+
+if {${os.platform} eq "darwin" && ${os.arch} eq "powerpc"} {
+    mpi.setup       require require_fortran \
+                    -gcc44 -gcc45 -gcc46 -gcc47 -gcc48 -gcc49 -gcc5 -gcc6 \
+                    -clang -fortran
+} else {
+    mpi.setup       require require_fortran \
+                    -gcc44 -gcc45 -gcc46 -gcc47 -gcc48 -gcc49 -gcc5 -gcc6
+}
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
+}
+
+compiler.cxx_standard   2014
+
+# Dynamic lib disabled for a reason: https://github.com/LLNL/RAJA/issues/1431
+# Tests also fail to build with similar linking errors.
+# OneTBB not yet supported: https://github.com/LLNL/RAJA/issues/1456
+configure.args-append \
+                    -DCMAKE_INSTALL_PREFIX=${prefix} \
```

Comment:
does the cmake PG not take care of this?

Reply:
I rebuilt it numerous times trying to get rid of Desul atomics, there was some weird behavior at destroot. But let me try removing and see if it works.

Reply:
You are right, thank you. My destroot troubles were caused by smth else. Removed.